scripts/timing.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+from collections import namedtuple
+import datetime
+import functools
+import os
+import random
+import string
+from subprocess import call
+
+RESULTS_BLOCKLTR = '''  ____                 _ _
+ |  _ \ ___  ___ _   _| | |_ ___ _
+ | |_) / _ \/ __| | | | | __/ __(_)
+ |  _ <  __/\__ \ |_| | | |_\__ \_
+ |_| \_\___||___/\__,_|_|\__|___(_)'''
+
+BLORGLY_BACKEND_DIR = os.path.join(os.environ['GOPATH'], 'src/github.com/windmilleng/blorgly-backend')
```

Comment:
not for this pr, but i wonder if this will eventually become a more OO class where the current backend dir and service names are fields of the class